packages/plugin-rename-all-fields/src/index.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type {
+  CollectionCustomizer,
+  DataSourceCustomizer,
+  Plugin,
+} from '@forestadmin/datasource-customizer';
+
+import { Options } from './types';
+
+export { Options as RenameAllFieldOption };
+
+export default function renameAllField(
+  dataSource: DataSourceCustomizer<any>,
+  collection: CollectionCustomizer<any>,
+  options?: Options,
+) {
+  if (!options) throw new Error('Options must be provided.');
```

Comment:
👀 `options?: Options` Should we remove the "?" from the type because we throw an error in this case.

Reply:
Agree but 
![image](https://github.com/user-attachments/assets/1c740388-2414-40bf-87c6-aaf1eaacb15f)